From dc6f7045665e1b543ec19cb27ddfe9173e35fb7c Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@z3ntu.xyz>
Date: Sun, 23 Apr 2023 20:09:22 +0200
Subject: [PATCH] freedreno: Enable A506

Enable the Adreno 506 that is found in SoCs such as Snapdragon 450,
Snapdragon 625 or Snapdragon 632 (msm8953 family).

Signed-off-by: Luca Weiss <luca@z3ntu.xyz>
Part-of: <https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/22647>
---
 src/freedreno/common/freedreno_devices.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/freedreno/common/freedreno_devices.py b/src/freedreno/common/freedreno_devices.py
index 628b2bf78c8..c8502d4edea 100644
--- a/src/freedreno/common/freedreno_devices.py
+++ b/src/freedreno/common/freedreno_devices.py
@@ -177,6 +177,7 @@ add_gpus([
     ))
 
 add_gpus([
+        GPUId(506),
         GPUId(508),
         GPUId(509),
         GPUId(510),
-- 
2.40.0

